build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
 		classpath 'io.spring.gradle:spring-build-conventions:0.0.23.RELEASE'
 		classpath "org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion"
 		classpath 'io.spring.nohttp:nohttp-gradle:0.0.2.RELEASE'
-		classpath "io.freefair.gradle:aspectj-plugin:4.0.1"
+		classpath "io.freefair.gradle:aspectj-plugin:4.0.2"
```

Comment:
@larsgrefer is this necessary in order to upgrade the Gradle version? 

To keep consistent with our team's pattern for doing upgrades, would you you please put this on a separate ticket?

Reply:
@jzheaux No, its not strictly necessary.

I'll split this into two PR's